{"name": "test_01", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 2", "trace": "@allure.severity('critical')\n    @allure.feature('模块3：xxxxx')\n    def test_01():\n        '''用例1的描述：xxxx'''\n        print('测试用例1')\n>       assert 1==2\nE       assert 1 == 2\n\ntest_x.py:13: AssertionError"}, "description": "用例1的描述：xxxx", "attachments": [{"name": "stdout", "source": "d6195b8f-e2d2-4c36-a834-9ad546d9b409-attachment.txt", "type": "text/plain"}], "start": 1595699192752, "stop": 1595699192753, "uuid": "4124b70b-b8b1-47be-be69-217adfdd6cbe", "historyId": "209a5198088eee57bf0ee8790dfb2514", "testCaseId": "8b287888448907456630b478095e9365", "fullName": "test_x#test_01", "labels": [{"name": "feature", "value": "模块3：xxxxx"}, {"name": "severity", "value": "critical"}, {"name": "suite", "value": "test_x"}, {"name": "host", "value": "kingving"}, {"name": "thread", "value": "16500-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_x"}]}